### PR TITLE
Add split-out fastsim signal samples

### PR DESCRIPTION
**Pull request by ahenckel**

### CODE REVIEW

**Comment by zhenbinwu** [comment]
Instead of commenting out the old line, can you remove them? Thanks

**Comment by ahenckel** [comment]
I would prefer to keep the information about the original preprocessed files available.

**Comment by zhenbinwu** [comment]
Ok, there are still some fastsim signal samples are commented out. Are they ready?

File: sampleSets_PreProcessed_2018.cfg [hunk 1]
```diff
@@ -178,34 +178,48 @@ SMS_T2tt_mStop_1200_mLSP_100_fullsim_2018, /eos/uscms/store/user/lpcsusyhad/Stop
...
+SMS_T2tt_mStop_1200to2000_fastsim_2018, /eos/uscms/store/user/lpcsusyhad/Stop_production/Autumn18_102X_v1/PreProcessed_26Apr2019_fastsimv5, SMS_T2tt_mStop_1200to2000_fastsim_2018.txt, Events, 1.0, 13199684, 0, 1.0
 # T5ttcc
-SMS_T5ttcc_fastsim_2018, /eos/uscms/store/user/lpcsusyhad/Stop_production/Autumn18_102X_v1/PreProcessed_26Apr2019, SMS-T5ttcc_TuneCP2_13TeV-madgraphMLM-pythia8.txt, Events, 1.0, 44077578, 0, 1.0
-
-SMS_T5ttcc_mGluino1750to2800_fastsim_2018, /eos/uscms/store/user/lpcsusyhad/Stop_production/Autumn18_102X_v1/PreProcessed_26Apr2019, SMS-T5ttcc_mGluino1750to2800_TuneCP2_13TeV-madgraphMLM-pythia8.txt, Events, 1, 17437129, 0, 1.0
+#SMS_T5ttcc_fastsim_2018, /eos/uscms/store/user/lpcsusyhad/Stop_production/Autumn18_102X_v1/PreProcessed_26Apr2019, SMS-T5ttcc_TuneCP2_13TeV-madgraphMLM-pythia8.txt, Events, 1.0, 44077578, 0, 1.0
+#SMS_T5ttcc_fastsim_2018, /eos/uscms/store/user/lpcsusyhad/Stop_production/Autumn18_102X_v1/PreProcessed_26Apr2019_fastsimv5, SMS_T5ttcc_fastsim_2018.txt, Events, 1.0, 44005488, 0, 1.0
```
Comment (zhenbinwu) [inline sampleSets_PreProcessed_2018.cfg]: Is this ready?

**Comment by zhenbinwu** [comment]
Ok. I am merging this PR for now in order to prep for V6. I hope some of the question can be updated before we start the fastsim signal.

**Pull request merged**